### DISCUSSION TITLE
Multiple notifications per type

### DISCUSSION THREAD

**Discussion by UrsKroell** [seerr-team/seerr#469] [Ideas]
Hello,

I'm in the process of setting up notifications for various services and during this I noticed that jellyseerr only supports one notification of each agent. So only one for email, discord, etc.

Would it be possible to add multiple of notifications of each agent?

As in, one email notification agent when a request needs approval, one when there are issues, etc. Since depending on the content there could be different requirements as the who should receive a notification.

edit: mixed up notification type and notification agent. I'm sorry.

**Comment by fallenbagel** [comment]
What do you mean? It should send multiple notification types.
It will send for approval, rejection, issues created, resolved, available.

![Screenshot_20230907_214804_Chrome Dev.jpg](https://github.com/Fallenbagel/jellyseerr/assets/98979876/f01259d1-f0c3-469f-940c-5cee9d2a650a)

![Screenshot_20230907_214746_Chrome Dev.jpg](https://github.com/Fallenbagel/jellyseerr/assets/98979876/034f1f8e-0a53-4516-9c65-c5e8d6d46a52)

**Comment by UrsKroell** [comment]
ah, sorry I mixed up notification types with notification agents.

**Comment by fallenbagel** [comment]
> ah, sorry I mixed up notification types with notification agents.

Multiple notification agents are already supported. I'm still not sure what you mean. Email is supported, web push is supported, discord is supported, gotify, lunasea, pushbullet, pushover, slack, telegram and custom webhooks

**Reply by UrsKroell** [reply]
yes, but not multiple of one kind of agent.

so only one email agent is supported at the moment.

**Reply by fallenbagel** [reply]
> yes, but not multiple of one kind of agent.
> 
> so only one email agent is supported at the moment.

OH I get it now. Well for the time being multiple notification for one agent can be handled by a custom webhook server (there are multiple webhook servers available selfhosted ones). So you would basically pass in all the variables in webhook agent, and let your webhook server route it to different agents depending on the type of notification etc

**Comment by angel-torres** [comment]
I know this is an old request but I agree. I'd like to, using Pushover for example, set my "Request Pending Approval" to go to one user/group key and the "Request Approved" or "Request Declined" go to a different user/group key. I'll try the webhook as a workaround.

**Reply by angel-torres** [reply]
Well, I just saw that each user has a Notification Settings section. It's in Users > Edit button > Notifications tab. The 3 options are Web Push, Pushbullet, and Pushover. As long you create an API key in Pushover and get the user key for the Pushover user,  you enter that in the Notification Settings for the user and they can get Approved, Declined, and Available messages. That's exactly what I was looking for. I use the main Pushover settings in Settings > Notification Settings to get the Pending Requests, Processing, and Issue messages.

**Comment by brad-charboneau** [comment]
I too would like to see support for multiple agents of the same type.   For Example, in Discord the ability to split Requests and Issues off into different channels or threads.

Or Requests and Availability Notifications to separate channels/threads.

**Reply by M0NsTeRRR** [reply]
There is an issue for that now :)
https://github.com/seerr-team/seerr/issues/804
